daemon/snap.go

```diff
@@ -160,6 +167,15 @@ func mapRemote(remoteSnap *snap.Info) map[string]interface{} {
 		confinement = snap.StrictConfinement
 	}
 
+	screenshots := make([]screenshotJSON, 0, len(remoteSnap.Screenshots))
```

Comment:
Clarifying what @mvo5 meant: if you avoid the 0 here, and add an `i` index in the iteration instead of `_`, you can simply do `screenshots[i] = ...`.